components/ErrorBanner/OldDesignatedVotingAccountWarningBanner.tsx

```diff
@@ -0,0 +1,72 @@
+import { useStakingContext } from "hooks";
+import NextLink from "next/link";
+import styled from "styled-components";
+import { useIsClient, useLocalStorage } from "usehooks-ts";
+import {
+  CloseButton,
+  CloseIcon,
+  ErrorMessage,
+  ErrorMessageWrapper,
+  IconWrapper,
+  WarningIcon,
+  Wrapper,
+} from "./styles";
+
+export function OldDesignatedVotingAccountWarningBanner() {
```

Comment:
component looks the same as the error banner, but it only deals with this particular situation